aggregator/src/app/AggregationStrategy.ts

```diff
@@ -231,22 +236,36 @@ export default class AggregationStrategy {
       assert(after.success);
 
       const bundleResult = processBundleResults[i];
-
-      let success: boolean;
-
-      if (bundleResult.success) {
-        const [operationResults] = bundleResult.returnValue;
-
-        // We require that at least one operation succeeds, even though
-        // processBundle doesn't revert in this case.
-        success = operationResults.some((opSuccess) => opSuccess === true);
-      } else {
-        success = false;
+      const fee = after.returnValue[0].sub(before.returnValue[0]);
+      if (!bundleResult.success) {
+        return { success: false, fee, errorReason: null };
```

Comment:
Looking through `AggregatorUtilities.sol` and the `EthereumService.ts` i think this makes a bit more sense now. Probably a larger/different discussion on how we are simulating actions.

Until then, I would put in something like `unknown` because that is the extent of what we know.

```suggestion
        return { success: false, fee, errorReason: "unknown" };
```